promql/engine.go

```diff
@@ -307,6 +334,36 @@ func NewEngine(opts EngineOpts) *Engine {
 	}
 }
 
+// UnsetQueryLogger disables the query logger.
+func (ng *Engine) UnsetQueryLogger() error {
```

Comment:
Is this function really needed? It's not really saving anything.

Reply:
removed